<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjoutProgramme</class>
 <widget class="QDialog" name="AjoutProgramme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>430</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>solde</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>90</y>
     <width>221</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="txt_nom">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="txt_solde">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>25</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBoxArticle"/>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Article</string>
   </property>
  </widget>
  <widget class="QLabel" name="titreLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>10</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Programme Ajout</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>nom</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
